BrewMesModulesParent/api/src/main/java/com/brewmes/api/BatchController.java

```diff
@@ -0,0 +1,67 @@
+package com.brewmes.api;
+
+import com.brewmes.common.entities.Batch;
+import com.brewmes.common.services.IBatchGetterService;
+import com.brewmes.common.services.IBatchReportService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/batch")
+public class BatchController {
+
+    @Autowired(required = false)
```

Comment:
Bør vi nok fjerne, mens vi husker det.
```suggestion
    @Autowired
```